src/webgpu/api/operation/labels.spec.ts

```diff
@@ -3,10 +3,271 @@ Tests for object labels.
 `;
 
 import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { getGPU } from '../../../common/util/navigator_gpu.js';
 import { GPUTest } from '../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
 
+type TestFunction = (t: GPUTest, label: string) => Promise<void>;
+const kTestFunctions: { [name: string]: TestFunction } = {
+  createBuffer: async (t: GPUTest, label: string) => {
+    const buffer = t.device.createBuffer({ size: 16, usage: GPUBufferUsage.COPY_DST, label });
+    t.expect(buffer.label === label);
+    buffer.destroy();
```

Comment:
just for fun, could check the label again after destroy(). Same for device/texture/queryset and the createView test

Reply:
done